docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+services:
+  pgduckdb:
+    container_name: pgduckdb
+    image: pgduckdb/pgduckdb:16-main
+    ports:
+      - ${PORT:-5432}:5432
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: ${PASSWORD:-duckdb}
+    volumes:
+      - pgduckdb_data:/var/lib/postgresql/data
+      - ./docker/postgresql.conf:/etc/postgresql/postgresql.conf
+      - ./docker/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
+    command: postgres -c 'config_file=/etc/postgresql/postgresql.conf'
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres -d postgres"]
+      interval: 2s
+      retries: 15
+      start_period: 30s
+      timeout: 2s
+    restart: always
+    networks:
+      - pgduckdb_net
+  psql:
+    image: postgres:16
+    depends_on:
+      pgduckdb:
+        condition: service_healthy
+    stdin_open: true
+    tty: true
+    environment:
+      PGPASSWORD: ${PASSWORD:-duckdb}
+    command: psql -h 127.0.0.1 -d postgres -U postgres -p ${PORT:-5432}
+    restart: always
+    networks:
+      - pgduckdb_net
```

Comment:
I think we can replace this with telling people to run:
```
docker compose exec pgduckdb psql -U postgres
```

Reply:
yeah, inclined to give up on this approach, though I still don't understand why it didn't work despite trying many different things (what I committed here was just the last version of what I tried).